java/parquet/pom.xml

```diff
@@ -95,12 +95,4 @@
         </dependency>
     </dependencies>
 
-    <build>
-        <plugins>
-            <plugin>
-                <groupId>org.apache.maven.plugins</groupId>
-                <artifactId>maven-shade-plugin</artifactId>
-            </plugin>
-        </plugins>
-    </build>
```

Comment:
This shading is present so that the `ShowPageIndexes` class can be run using the shaded jar.

Reply:
That might work better